### PR TITLE
Create controller fixes #3

### LINKED ISSUE #2: Add middleware support
Till last commit, it lacks middleware support. Goal is to establish middleware on Torn application through command line.

``` php
if(Auth::user())
        {
            $role = Auth::user()->role;
            if($role==1)
            {
                return $next($request);
            }
            else{
                return view('errors.403');
            }
        }
```

Just like Laravel's authentication middleware to allow admin users.

### LINKED ISSUE #3: Create a new controller from command line
Something like `torn make:controller controllerName`

### PR DESCRIPTION
**Pull request by ashwini0529**
- Added a file `createController.py` that handles file create operations.
- In `torn.py` , the argument structure for creating a controller is proposed as : 

> `torn create controller ControllerName`

This will create a controller in the `controllers` directory of the project.

### CODE REVIEW

File: createController.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import os
+from colors import bcolors
+class createNewController():
+	NAME = ''
+	CURRENT_DIR = os.getcwd()
```
Comment (shubhodeep9) [inline createController.py]: Rest is fine, kindly make directory fetching local, instead of global

**Comment by shubhodeep9** [comment]
@ashwini0529 see to the comment I posted on createController.py, and kindly give some helping comments.

File: createController.py (hunk 1 above)
Comment (ashwini0529) [reply createController.py]: Alright.👍

**Comment by ashwini0529** [comment]
@shubhodeep9, please I've added some commits. Please look into it.

**Comment by shubhodeep9** [comment]
The concept is still incomplete, right now all it is doing is creating a file, which easily can be established using just a touch command in bash.
I'd recommend going through Sails.js create controller concept.

**Pull request merged**